ingestion/functions/Dockerfile

```diff
@@ -1,13 +1,72 @@
-FROM python:3.8  
+# `python-base` sets up all our shared environment variables
+FROM python:3.8.1-slim as python-base
```

Comment:
Is there a 3.8-slim? Might as well get the point updates automatically

Reply:
Good point. Updated.